client/landing/gutenboarding/stores/onboard/actions.ts

```diff
@@ -74,6 +74,7 @@ export function* createSite(
 				font_headings: selectedFonts.headings,
 			} ),
 			use_patterns: isEnabled( 'gutenboarding/use-patterns' ),
+			public_coming_soon: isEnabled( 'gutenboarding/public-coming-soon' ) || undefined,
```

Comment:
this will need to be updated to the new flag name `wpcom_public_coming_soon`.

Is the `|| undefined` necessary? that would be the same behavior as leaving it off wouldn't it?

Reply:
> this will need to be updated to the new flag name `wpcom_public_coming_soon`.

This is an API param so doesn't actually need to map directly to the site option. But maybe leaving out the `wpcom_` when the name is so similar just makes it confusing.

> Is the `|| undefined` necessary?

It might not be. I wanted to leave out the param from the payload if it's disabled, but I didn't actually test that the `|| undefined` was necessary.

Reply:
Just to be clear sending `public_coming_soon=false` would still do the correct thing, I kinda just wanted to leave it off entirely though.

Reply:
I saw the existing code was doing the `...( someBool && { ... } )` trick to leave out some options. So I switched to that instead.

Reply:
ahh ok I see why it's neccessary 👍